example/src/App.tsx

```diff
@@ -31,6 +32,7 @@ export type RootStackParamsList = {
   BasicAds: undefined;
   BasicAnalytics: undefined;
   BasicPlayback: undefined;
+  NoUiSubtitlesOnly: undefined;
```

Comment:
I have no strong opinion on that. :+1: 
I added it to have the change testable.

whats your take on this @matamegger

Reply:
I don't see a particular need, but I have no strong opinion on that.

It would align with a similar samples from the android player, if we also add a react player controls UI on top of that 😄 but that's out of scope for this PR I guess.

I think it's fine if there is not sample for this.

Reply:
removed the example: https://github.com/bitmovin/bitmovin-player-react-native/pull/316/commits/942d9b0e98e4a24a89ba0cc265c91fbe747f3917